includes/class-wc-gateway-komoju-request.php

```diff
@@ -20,63 +20,52 @@ class WC_Gateway_Komoju_Request {
 	 */
 	protected $gateway;
 
-	/**
-	 * Endpoint for requests from Komoju
-	 * @var string
-	 */
-	protected $notify_url;
-
 	/**
 	 * Constructor
 	 * @param WC_Gateway_Komoju $gateway
 	 */
 	public function __construct( $gateway ) {
 		$this->gateway    = $gateway;
-		$this->notify_url = $this->gateway->notify_url;

```

Comment:
the `$notify_url` is the callback URL, which has been deprecated and replaced with the webhook events.